addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -861,7 +861,8 @@ const SelectUserValueWidget = BaseSelectionUserValueWidget.extend({
         let textContent = '';
         const activeWidget = this._userValueWidgets.find(widget => !widget.isPreviewed() && widget.isActive());
         if (activeWidget) {
-            const value = (activeWidget.el.dataset.selectLabel || activeWidget.el.textContent.trim());
+            const svgTag = activeWidget.$el.find('svg').length ? true : false; // usefull to avoid searching text content in svg element
```

Comment:
```suggestion
            const svgTag = !!activeWidget.$el.find('svg').length; // useful to avoid searching text content in svg element
```
or even
```suggestion
            const svgTag = activeWidget.el.querySelector('svg'); // useful to avoid searching text content in svg element
```

Reply:
:+1:  Done